lpl_ca/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ target_include_directories(ca PUBLIC ${PROJECT_SOURCE_DIR}/src)
 find_library(
     hsart NAMES libhsa-runtime64.so libhsa-runtime64.so.1 HINTS ${HSA_PATH}/lib)
 target_link_libraries(ca PUBLIC ${hsart})
-target_compile_options(ca PUBLIC -Wall)
+target_compile_options(ca PUBLIC -DDISABLE_REDUCED_GPU_BLOB_COPY -Wall)
```

Comment:
Should we remove this? Otherwise this patch is disabled and will not get tested.

Reply:
This is disabled for lpl_ca . It is enabled for HIP runtime.

Reply:
reason?

Reply:
lpl_ca is used to generate HSA code objects. Reduced GPU copy is not applicable. and get_all_gpuarch() defined in hip_hcc.cpp is not visible for lpl_ca and hiprtc.

Reply:
>lpl_ca is used to generate HSA code objects. Reduced GPU copy is not applicable.
This is speculative.  Note that the interface is a vector of targets https://github.com/ROCm-Developer-Tools/HIP/blob/master/lpl_ca/lpl.hpp#L67
 
>get_all_gpuarch() defined in hip_hcc.cpp is not visible for lpl_ca and hiprtc.
can we fix that?

Reply:
lpl/ca generate hsaco fat binaries that are stored to disk for use in ```hipModuleLoad()``` on the actual target system. So lpl/ca should not drop GPU targets based on hardware on the build system. Hence ```get_all_gpuarch()``` is not applicable to lpl/ca. Also ```hipModuleLoad()``` would drop the GPU targets when we load the hsaco fat binary in the target system.